frontend/src/components/passenger/PassengerWorkspace.tsx

```
@@ -0,0 +1,592 @@
+      trackedRideRef.current,
+      activeRides,
+      (sessionId) => latestTripStatesRef.current.get(sessionId),
+    );
+
+    switch (action.type) {
+      case "complete":
+        trackedRideRef.current = null;
+        const completionTimer = setTimeout(() => {
+          setTrackedSessionId("");
+          setCompletedRide(action.ride);
+        }, 0);
```

Biome (2.5.6)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)